### PR TITLE
Remove nodejs

### PR DESCRIPTION
**Pull request by swilen-iwanow**
**What this PR does / why we need it**:
Removes the dependency to Node.js as the previous scripts written in javascript are now replaced entirely by Docforge. 
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator

```

### CODE REVIEW

File: .ci/build [hunk 1]
```diff
@@ -29,33 +28,16 @@ fi
...
-  export BUILDSINGLEBRANCH=true
+if [ ! -f "./latestDocVersions" ]; then
+  exit -1
 fi
 
-if [[ -z "${FORK}" ]]; then
-  export FORK=gardener
-fi
+versions=$(cat ./latestDocVersions)
+export LATESTVERSIONS=($versions) # make it arr
```
Comment (g-pavlov) [inline .ci/build:40]: it seems we are not using these

File: .ci/build [hunk 2]
```diff
@@ -29,33 +28,16 @@ fi
...
-# versioned.
-if [[ -z "${LATESTVERSIONS}" ]]; then
-  if [[ -z "${BRANCH}" ]]; then
-    export BRANCH=master
-  fi
-  echo "Building from single branch ${BRANCH}"
-  export BUILDSINGLEBRANCH=true
+if [ ! -f "./latestDocVersions" ]; then
+  exit -1
 fi
```
Comment (g-pavlov) [inline .ci/build:37]: is this necessary?

File: .ci/build [hunk 3]
```diff
@@ -94,55 +71,9 @@ if [[ $BRANCH != "master" && ! -z $BRANCH ]]; then
...
-    fi
-
-    cp -r $CLONED_DOCUMENTATION_PATH $HUGO_CONTENT/$version
-    docforge -f ${CLONE}/documentation.yaml -d $HUGO_CONTENT/$version --hugo --github-oauth-token $GIT_OAUTH_TOKEN
-    node ./node/index.js
-  done
-  node ./node/generateVersioningFile.js
-fi
-
+docforge -f ${CLONE}/.docforge/website.yaml -d $HUGO_CONTENT --resources-download-path __resources --hugo --github-oauth-token $GIT_OAUTH_TOKEN --github-info-destination ../data || true
```
Comment (g-pavlov) [inline .ci/build:74]: The `__resources` value is the default for the `--resources-download-path ` flag so not strictly necessary here. 
```suggestion
docforge -f ${CLONE}/.docforge/website.yaml -d $HUGO_CONTENT --hugo --github-oauth-token $GIT_OAUTH_TOKEN --github-info-destination ../data || true
```

**Review by g-pavlov — CHANGES_REQUESTED** [review]

**Comment by swilen-iwanow** [comment]
@g-pavlov ptal.

**Review by g-pavlov — APPROVED** [review]
tal
/lgtm

**Pull request merged**